HomeTask8/app/src/app/auth/auth.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { NewsService } from '../services/news.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+loggedInUserName : string;
+  constructor(private newsService : NewsService) { 
+  }
+
+  get showArticle(): boolean {
+    this.loggedInUserName = this.newsService.getLoggedInUserName;
```

Comment:
can be rewritten as retrun this.newsService.getLoggedInUserName === 'admin'

Reply:
Done.